packages/next-polyfill-nomodule/src/index.js

```diff
@@ -11,6 +11,9 @@ import 'core-js/features/array/from'
 import 'core-js/features/array/includes'
 import 'core-js/features/array/iterator'
 import 'core-js/features/array/of'
+import 'core-js/features/array/to-reversed'
+import 'core-js/features/array/to-spliced'
+import 'core-js/features/array/to-sorted'
```

Comment:
@MonstraG You should also add `with` polyfill 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/with
https://medium.com/@jemimaosoro/ecmascript-2023-mastering-tosorted-toreversed-tospliced-and-with-array-methods-61030c57a677

 ```suggestion
import 'core-js/features/array/to-sorted'
import 'core-js/features/array/with'
```